Apply black styling to test_align*.py

jvfe opened the pull request:
This pull request addresses issue #2552 

<!--- Please read each of the following items and confirm by replacing
 !--the [ ] with a [X] --->

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

peterjc commented on Tests/test_align_substitution_matrices.py:
```diff
@@ -120,7 +145,7 @@ def test_nucleotide_freq(self):
         frequencies = substitution_matrices.read(path)
         self.assertEqual(frequencies.alphabet, nucleotide_alphabet)
-        self.assertEqual(frequencies.shape, (len(nucleotide_alphabet), ))
+        self.assertEqual(frequencies.shape, (len(nucleotide_alphabet),))
```
I think that final comma can be removed (here and 3 more cases lower down - very obvious looking at the diff on GitHub).

jvfe replied:
I believe I got all of them.

peterjc replied:
As noted later, these were deliberate for expressing a single element tuple

peterjc commented:
Oh - I misread the comma placement, it makes sense now:

```
======================================================================
FAIL: test_nucleotide_freq (test_align_substitution_matrices.Test_basics)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/biopython/biopython/Tests/test_align_substitution_matrices.py", line 148, in test_nucleotide_freq
    self.assertEqual(frequencies.shape, (len(nucleotide_alphabet)))
AssertionError: (4,) != 4
======================================================================
FAIL: test_protein_freq (test_align_substitution_matrices.Test_basics)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/biopython/biopython/Tests/test_align_substitution_matrices.py", line 202, in test_protein_freq
    self.assertEqual(frequencies.shape, (len(protein_alphabet)))
AssertionError: (20,) != 20
----------------------------------------------------------------------
```

The comma was deliberate to express a single element tuple. There's a Python quirk you don't see everyday.

Easily fixed, ``git revert 9729098a244c3d3aaef8a1528f4f5ebfc152423c`` to undo the commit.

peterjc commented:
Or, since this is a pull request and we don't mind rewriting the commit history specific to this branch, a nice chance to try ``git rebase master -i`` for an interactive rebase - and comment out the ``9729098a244c3d3aaef8a1528f4f5ebfc152423c`` line. Then you'd need to to push the changes and this time ``git push ... --force`` which is one of the few occasions you'd force a push and overwrite the branch history (rather than adding to it).

(Apologies if this is all familiar to you already - seemed like a good learning opportunity)

peterjc commented:
Sorry, deleted my earlier comment. Most of the comma edits seem fine - just two of them were deliberate to express a single element tuple.

i.e. We don't want to revert the entire second commit - since we want to keep some of the edits.

jvfe commented:
> Or, since this is a pull request and we don't mind rewriting the commit history specific to this branch, a nice chance to try `git rebase master -i` for an interactive rebase - and comment out the `9729098a244c3d3aaef8a1528f4f5ebfc152423c` line. Then you'd need to to push the changes and this time `git push ... --force` which is one of the few occasions you'd force a push and overwrite the branch history (rather than adding to it).
> 
> (Apologies if this is all familiar to you already - seemed like a good learning opportunity)

That's something I didn't know, actually! Used revert before but never rebasing and force pushing, thanks for the tip!



> Sorry, deleted my earlier comment. Most of the comma edits seem fine - just two of them were deliberate to express a single element tuple.
> 
> i.e. We don't want to revert the entire second commit - since we want to keep some of the edits.

Unfortunately, before I saw your comment I had already reverted the commit, but I can make another commit. 

Just to be clear, the lines that must have the commas are [148](https://github.com/biopython/biopython/blob/9122f3c12e0dbddcc9d971111df256850dba4e4e/Tests/test_align_substitution_matrices.py#L148) and [202](https://github.com/biopython/biopython/blob/9122f3c12e0dbddcc9d971111df256850dba4e4e/Tests/test_align_substitution_matrices.py#L202), since they're supposed express tuples, right?

peterjc commented:
If you're learning more about git, that is a good outcome. And yes, the failed tests show us that the comma was important in ``test_nucleotide_freq`` (line 148) and ``test_protein_freq`` (line 202).

jvfe commented:
> If you're learning more about git, that is a good outcome. And yes, the failed tests show us that the comma was important in `test_nucleotide_freq` (line 148) and `test_protein_freq` (line 202).

Found out the other trailing commas (L173 and L243) are also necessary, as I ran the tests for this file - something that escaped me the last time around - and got the following output:

```
======================================================================
FAIL: test_nucleotide_freq (test_align_substitution_matrices.Test_basics)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "--", line 173, in test_nucleotide_freq
    self.assertEqual(frequencies.shape, (len(nucleotide_alphabet)))
AssertionError: (4,) != 4
======================================================================
FAIL: test_protein_freq (test_align_substitution_matrices.Test_basics)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "--", line 243, in test_protein_freq
    self.assertEqual(frequencies.shape, (len(protein_alphabet)))
AssertionError: (20,) != 20

----------------------------------------------------------------------
Ran 1 test in 1.422 seconds
```
(Just removed my system's paths for clarity)

Tests only passed when I left all the commas in, so in that case I think I should leave it as is.

peterjc commented:
Oops. So in my hasty review I made some extra work, but if you learnt a bit about git and reverting commits, that's a useful outcome. Thank you!
